src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,12 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] inProgress = text.replaceAll("[^-?0-9]+", " ").trim().split("(\\s+)");
```

Comment:
What means `inProgress`. This is a bad variable name

Reply:
done